tests.py

```diff
@@ -16,7 +16,7 @@
 from typing import Optional
 from xml.etree import ElementTree as ET
 
-import mock
+import unittest.mock as mock
```

Comment:
PEP-8 rules for import ordering: stdlib imports first, 3rd-party imports second, local imports last, with a blank line separating the sections.

This replaces a 3rd-party import with a stdlib import, so the location needs to change.  Also, please keep the imports sorted alphabetically.

I have [isort](https://pypi.org/project/isort/) rules in setup.cfg, so if you run `isort` on the file, it'll do the right thing.

(Also, this could be `from unittest import mock`, it'd be a bit shorter.)